urllib3/connectionpool.py

```diff
@@ -10,7 +10,8 @@
     from queue import LifoQueue, Empty, Full
 except ImportError:
     from Queue import LifoQueue, Empty, Full
-    import Queue as _  # Platform-specific: Windows
+    # Queue is imported for side effects on MS Windows
+    import Queue as _unused_module_Queue  # noqa: unused
```

Comment:
note that just `_` meant that on line 186 the variable `_` in `for _ in xrange(maxsize):` was shadowing this symbol.